src/main/java/app/service/impl/OperationReturn.java

```diff
@@ -0,0 +1,31 @@
+package app.service.impl;
+
+import app.FruitStorage;
+import app.model.SupplyFruit;
+import app.service.Operation;
+import java.util.List;
+
+public class OperationReturn implements Operation {
+    @Override
+    public void doOperation(List<String> data) {
+        SupplyFruit currentFruit = new FruitParserImplements().parse(data);
+        boolean isExist = false;
```

Comment:
boolean is false by default

Reply:
Of course, but we have to initialize variable to use it in line 19.